tensorflow_addons/optimizers/moving_average.py

```diff
@@ -18,10 +18,13 @@
 from __future__ import print_function
 
 import tensorflow as tf
+from tensorflow.python.training.moving_averages import assign_moving_average
```

Comment:
`assign_moving_average` is currently not exposed via a public API. The implementation is simple enough to include directly into `tfa`, but the upstream `tf` implementation has support for averaging under a distribution strategy which would be desirable to have.

Reply:
Created an issue to make `assign_moving_average` public: https://github.com/tensorflow/tensorflow/issues/35118